src/w3iProxy/pushProviders/internalPushProvider.ts

```diff
@@ -124,9 +126,18 @@ export default class InternalPushProvider implements W3iPushProvider {
     if (!this.pushClient) {
       throw new Error(this.formatClientRelatedError('subscribe'))
     }
-
     console.log('InternalPushProvider > PushClient.subscribe > params', params)
 
+    Notification.requestPermission()
+
+    const clientId = await this.pushClient.core.crypto.getClientId()
+
+    // Retrieving FCM token needs to be client side, outside the service worker.
+    const token = await getToken(messaging, {
+      vapidKey:
+        'BCnI0mkpH3LvHRF-dREPCdvBFk24oveWy4JBuINzWcu8JXhmCDkczDmHM9RubzsQrv60UKFk-MKozVjRRzvx1X4'
```

Comment:
Can we avoid hard coding this inline and use an env var or add it into the firebase config instead pls?